pages/linux/debsecan.md

```diff
@@ -0,0 +1,24 @@
+# debsecan
+
+> A tool to list vulnerabilities on a particular Debian installation. 
+> More information: <https://gitlab.com/fweimer/debsecan>.
+
+- List vulnerable installed packages on the current host:
+
+`debsecan`
+
+- List vulnerable installed packages of a specific suite:
+
+`debsecan --suite {{release_code_name}}`
+
+- List only fixed vulnerabilities of testing ("sid"):
+
+`debsecan --suite sid --only-fixed`
```

Comment:
@marchersimon  It's supposed to be `debsecan --suite {{sid}} --only-fixed` or just leave it as it was?

Reply:
You're right, I didn't see it.
```suggestion
`debsecan --suite {{sid}} --only-fixed`
```